pkg/cmd/root.go

```diff
@@ -37,6 +38,7 @@ func Root() *cobra.Command {
 	cmd.Flags().Bool(flagTrimLines, true, "trim any trailing spaces from each line")
 	cmd.Flags().Bool(flagLint, false, "don't modify files, but exit 1 if files aren't formatted")
 	cmd.Flags().StringP(flagConfig, "c", "", "path to a yam configuration YAML file")
+	cmd.Flags().StringSlice(flagQuote, nil, "YAML path expression to a node or sequnce of nodes that should be quoted")
```

Comment:
`sequnce of nodes`: I think using "sequence" is probably confusing here, since elsewhere in these flag descriptions we're referring to "sequence" the YAML AST concept.

Maybe just this would suffice?

```suggestion
	cmd.Flags().StringSlice(flagQuote, nil, "YAML path expression to a node that should be quoted")
```

(We refer to these nodes singularly for the other `StringSlice` flags)